### PR TITLE
Jamie/bug table widths

### PR DESCRIPTION
**Pull request by SEAjamieD**
### :nut_and_bolt: Description: What code changed, and why?
Long content in tables is causing layout issues

### :+1: Definition of Done
Content is ellipsed and no longer breaks layout/looks the same as it did prior to having long content in it.

### :athletic_shoe: How to Build and Test the Change
build components/automate-ui-devproxy && start_all_services

cd components/automate-ui
make serve

To Test
navigate to https://a2-dev.test/settings/tokens
create a token with a really long name
observe that the long names in the api token list are now ellipsed instead of affecting layout

### :white_check_mark: Checklist

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated?
- [ ] Docs added/updated?
- [ ] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### :camera: Screenshots, if applicable
Before

broken

Screen Shot 2020-02-25 at 4 47 27 PM

### CODE REVIEW

**Comment by SEAjamieD** [comment]
Whoops...this is currently a draft - please hold on reviews

File: components/automate-ui/src/app/pages/+compliance/+profile/+profile-overview/profile-overview.component.html [hunk 1]
```diff
@@ -199,7 +199,7 @@ <h2 id="upload-modal-title" slot="title">Upload an archived profile (.tar.gz or
                   </a>
                 </chef-table-cell>
                 <chef-table-cell>{{ profile.version }}</chef-table-cell>
-                <chef-table-cell>
+                <chef-table-cell class="text-right">
```
Comment (msorens) [inline components/automate-ui/src/app/pages/+compliance/+profile/+profile-overview/profile-overview.component.html]: Slight concern over why this table is unique...?

File: components/automate-ui/src/app/components/table/table-cell/table-cell.component.scss [hunk 2]
```diff
@@ -1,12 +1,19 @@
 @import '../table.component';
 
 :host {
```
Comment (msorens) [inline components/automate-ui/src/app/components/table/table-cell/table-cell.component.scss]: These changes fix all the "list of x" pages.

File: components/automate-ui/src/app/modules/policy/details/policy-details.component.scss [hunk 3]
```diff
@@ -40,9 +40,12 @@ tbody > .detail-row {
 }
 
 .id-column {
```
Comment (msorens) [inline components/automate-ui/src/app/modules/policy/details/policy-details.component.scss]: Each of the "x-details" pages that shows a one-line table of the item needs these changes individually, so you will see this pattern applied in several files.
I only checked the auth-pages, though; are there others...?

File: components/automate-ui/src/app/pages/+compliance/+profile/+profile-overview/profile-overview.component.html (hunk 1 above)
Comment (SEAjamieD) [reply components/automate-ui/src/app/pages/+compliance/+profile/+profile-overview/profile-overview.component.html]: in order to make the ellipses apply, content needs to be `inline-block`, which then caused - in this elements case - it to no longer be right aligned.  A simple and low code fix is to use `text-align: right` which I wanted to be reusable incase it needed to be used elsewhere.
The reason it was happening only here (that I know of yet) is that most of the columns contain text, where as this column contain an icon so it was thrown off in a way the others weren't

**Pull request merged**